README.md

```diff
@@ -466,6 +445,8 @@ Translations of the guide are available in the following languages:
     good - leading `.` (Option A) and trailing `.` (Option B).
 <sup>[[link](#consistent-multi-line-chains)]</sup>
 
+At Goldstar, we use (Option B)
```

Comment:
FWIW - Option(A) will indent properly in vim, while Option(B) will not, as I discovered today to my chagrin.